common/subdevice-model.h

```diff
@@ -207,6 +207,7 @@ namespace rs2
         std::vector<std::shared_ptr<processing_block_model>> const_effects;
 
         bool uvmapping_calib_full = false;
+        hdr_model _hdr_model;
```

Comment:
Don't add public members. Should be private and declare public functions that hide implementation details. e.g. subdevice_model::supports_hdr()